Cargo.toml

```diff
@@ -15,9 +15,14 @@ repository = "https://github.com/ebfull/pairing"
 rand = "0.4"
 byteorder = "1"
 clippy = { version = "0.0.200", optional = true }
+serde = { version = "1.0.66", optional = true }
```

Comment:
Why not `1.0`?

Reply:
I'll change it.

("`1.0.66`" has the same meaning as "`1.0`", except that it excludes versions earlier than `1.0.66`. I usually write it that way because I haven't tried it with versions earlier than when I introduced the dependency, and I don't know how any of the bugs fixed since then would affect the crate.)